libosmscout/include/osmscout/util/utf8helper_charmap.h

```diff
@@ -0,0 +1,78 @@
+#ifndef UTF8HELPER_CHARMAP_H
+#define UTF8HELPER_CHARMAP_H
+
+namespace utf8helper
+{
+
+constexpr char None         = 0x00;
+constexpr char IsSpace      = 0x01;
+constexpr char IsBreaker    = 0x02;
+constexpr char IsControl    = 0x04;
+
+using byte = unsigned char;
+
+using codepoint = unsigned int;
```

Comment:
Shouldn't you not better use something like uint8_t...uint32_t?

Reply:
Yes It would be better. It is an old habit to avoid compiler failure with old platforms. To be fixed including stdint.